usaspending_api/broker/helpers/last_delta_table_load_version.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from usaspending_api.broker import lookups
+from usaspending_api.broker.models import DeltaTableLoadVersion
+
+logger = logging.getLogger("script")
+
+
+def get_last_delta_table_load_versions(key):
+    # TODO - Pydoc
+    delta_table_load_version_id = lookups.LOAD_VERSION_TYPE_DICT[key]
+
+    last_version_to_staging, last_version_to_live = (
+        DeltaTableLoadVersion.objects.filter(delta_table_load_version_id=delta_table_load_version_id)
+        .values_list("last_version_copied_to_staging", "last_version_copied_to_live")
+        .first()
+    )
+
+    if not last_version_to_staging or not last_version_to_live:
+        logger.error(f"No record of previous run for key `{key}` was found!")
+        # TODO - Throw an error here
+    else:
+        logger.info(f"Value for `{key}` last_version_copied_to_staging: {last_version_to_staging}")
+        logger.info(f"Value for `{key}` last_version_copied_to_live   : {last_version_to_live}")
+
+    return last_version_to_staging, last_version_to_live
+
+
+def update_last_live_load_version(key, version):
+    DeltaTableLoadVersion.objects.update_or_create(name=key, defaults={"last_version_copied_to_live": version})
+
+
+def update_last_staging_load_version(key, version):
+    DeltaTableLoadVersion.objects.update_or_create(
+        delta_table_load_version_id=lookups.LOAD_VERSION_TYPE_DICT[key],
```

Comment:
Same thing as above,I think it would be good to either:
* Use `lookups.LOAD_VERSION_TYPE_DICT.get(key)` to prevent a possible `KeyError`.
* Catch the possible `KeyError` if you want keep the current indexing code.Az

Reply:
I think throwing a KeyError here should be the intended behavior. Not really a way to gracefully recover from a key missing from the lookup.